env.local.example

```diff
@@ -1,2 +1,2 @@
-MONGO_URI=< LINK TO MLAB INSTANCE HERE >
-JWT_SECRET=< JWT SECRET HERE >
\ No newline at end of file
+MONGO_URI=mongodb://bear:str0nkbear@ds127704.mlab.com:27704/habit-app
```

Comment:
These details shouldn't be committed. This is just an example file, the actual `env.local` is in `.gitignore` so that this is not made public.

Reply:
![:(](https://vignette.wikia.nocookie.net/cardfight/images/3/3a/Darth_vader_noooo.jpg/revision/20160303002440)